src/ir_Daikin.h

```diff
@@ -110,9 +112,11 @@ union DaikinESPProtocol{
     uint64_t          :4;
     uint64_t Quiet    :1;
     uint64_t          :2;
-    // Byte 30~31
-    uint64_t          :0;
-
+    // Byte 30
+    uint64_t          :8;
+    // Byte 31
+    uint64_t Arc484a4 :1;  // Always `1` for ARC484A4 models. Ref: Issue#1552
```

Comment:
Will GitHub create a link if a space is added ? "Ref: Issue #1552"

Reply:
Probably not, but it's just a pointer to where it came from. So, meh. I'm not worried if it doesn't make a link.